01-path-basics/102-your-first-cluster/readme.adoc

```diff
@@ -174,206 +141,56 @@ The next step is link:../103-kubernetes-concepts[to learn about basic Kubernetes
-
-If you created a private VPC, then an additional cleanup of resources is required as shown below:
-
-    # Find Route53 hosted zone ID from the console or via CLI and delete hosted zone
-    aws route53 delete-hosted-zone --id $ZONEID
-    # Delete VPC if you created earlier
-    $ aws ec2 detach-internet-gateway --internet $IGW --vpc $VPCID
-    aws ec2 delete-internet-gateway --internet-gateway-id $IGW
-    aws ec2 delete-vpc --vpc-id $VPCID
-
-To remove the state store S3 bucket:
-
-    aws s3 rb $KOPS_STATE_STORE
+This section will walk you through how to install a Kubernetes cluster on AWS using kops.
 
-== Appendix: Create a DNS-based Kubernetes cluster
+https://github.com/kubernetes/kops[kops, window="_blank"], short for Kubernetes Operations, is a set of tools for installing, operating, and deleting Kubernetes clusters. kops can also perform rolling upgrades from older versions of Kubernetes to newer ones, and manage the cluster add-ons.
 
-To create a DNS-based Kubernetes cluster you'll need a top-level domain or subdomain that meets one of the following scenarios:
+kops can be used to create a highly available cluster, with multiple master and worker nodes spread across multiple availability zones.
+The master and worker nodes within the cluster can use either DNS or the https://github.com/weaveworks/mesh[Weave Mesh, window="_blank"] *gossip* protocol for name resolution.  For this workshop, we will use the gossip protocol.  A gossip-based cluster is easier and quicker to setup, and does not require a domain, subdomain, or Route53 hosted zone to be registered. Instructions for creating a DNS-based cluster are provided as an appendix at the bottom of this page.
 
-. Domain purchased/hosted via AWS
-. A subdomain under a domain purchased/hosted via AWS
-. Setting up Route53 for a domain purchased with another registrar, transfering the domain to Route53
-. Subdomain for clusters in Route53, leaving the domain at another registrar
+To create a cluster using the gossip protocol, simply use a cluster name with a suffix of `.k8s.local`. In the following steps, we will use `example.cluster.k8s.local` as a sample gossip cluster name. You may choose a different name as long as it ends with `.k8s.local`.
 
-Then you need to follow the instructions in https://github.com/kubernetes/kops/blob/master/docs/aws.md#configure-dns[configure DNS]. Typically, the first and the last bullets are common scenarios.
+We show two examples of creating gossip-based clusters below. You can choose whether to create a single-master or multi-master cluster. Most workshop exercises will work on both types of cluster, however some modules require using a multi-master cluster _(to demonstrate rolling updates, for instance)._ If you aren't sure, please create a xref:multi-master[multi-master cluster].
 
-==== Single-Master DNS-based cluster
+=== Single-Master Cluster
```

Comment:
Lets keep only the multi-master option to keep it par with EKS?

Reply:
Done.